tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -1610,6 +1615,57 @@ async def test_add_nodes(self, bft_network):
             nb_fast_path = await bft_network.get_metric(r, bft_network, "Counters", "totalFastPaths")
             self.assertGreater(nb_fast_path, 0)
 
+
+    @with_trio
+    @with_bft_network(start_replica_cmd, bft_configs=[BFTConfig(n=4, f=1, c=0)], publish_master_keys=True)
+    async def test_key_exchange_after_add_nodes_with_failure(self, bft_network):
```

Comment:
cant we add it as part of the add_nodes_with_failure (and save the test time)?

Reply:
Also, maybe I'm missing something, but I don't see that there are failed replicas in this test

Reply:
1. add_nodes_with_failure needs to be shortened if anything.
2. This is a simpler (and faster) scenario where the network is scaled up, two replicas are behind, and they are expected to complete ST successfully one by one and exchange their keys.